ui/js/signup.js

```diff
@@ -319,6 +319,53 @@ require(['jquery', 'underscore', 'oae.core'], function($, _, oae) {
     };
 
     /**
+     * Initialize the signup institution search template
+     */
+    var renderSignUpInstitutionSearch = function() {
+        if (!oae.data.me.tenant.isGuestTenant) {
+            // Don't render institution search at all if we're not on the guest
+            // tenant
+            return;
+        }
+
+        oae.api.util.template().render($('#signup-institution-template'),
+            null, $('#signup-institution-container'));
+
+        oae.api.util.autoSuggest().setup($('#signup-institution-search-field'), {
+            'url': '/api/search/tenants',
+            'formatList': function(data, $el) {
+                var countryInfo = _.findWhere(iso3166.countries, {'code': data.countryCode});
+                oae.api.util.template().render($('#signup-autosuggest-tenant-template'), {
+                    'tenant': _.extend({}, data, {
+                        'countryName': (countryInfo && countryInfo.name),
+                        'countryFlag': (countryInfo && countryInfo.icon)
+                    })
+                }, $el);
+                return $el;
+            },
+            'selectionAdded': function(el) {
```

Comment:
http://a9yrj6.axshare.com/home.html

You have to type in “uni” to trigger auto suggest and only Aberdeen works